docs/guides/build-container-image.md

```diff
@@ -31,11 +31,11 @@ Dockerfile
 ```dockerfile
 FROM alpine:3.16
 RUN apk add curl
-RUN curl -sSfL https://raw.githubusercontent.com/aquaproj/aqua-installer/v3.0.0/aqua-installer | sh -s -- -i /usr/local/bin/aqua -v v2.25.1
+ENV AQUA_ROOT_DIR=/usr/local
```

Comment:
I don't think this is good.
aqua creates the following files in AQUA_ROOT_DIR.

```console
$ ls ~/.local/share/aquaproj-aqua 
aqua-proxy  bin  internal  pkgs  policies  registries
```

I don't think these files should be created on `/usr/local`.

Reply:
I see. So, would it be safer not to set`AQUA_ROOT_DIR`and instead set `PATH="/root/.local/share/aquaproj-aqua/bin:$PATH"`? Or is there a better approach?

Reply:
I think `PATH="/root/.local/share/aquaproj-aqua/bin:$PATH"` is better.